src/components/map/MapComponent.vue

```diff
@@ -183,34 +179,35 @@ travelTimesViewStore.$subscribe(async () => {
 })
 
 stationsStore.$subscribe(async () => {
-  await updateMapStyle()
-  await componentAboveMapStore.updateListLabelsStations(rennesApp)
+  await updateTraveltimeArrow(rennesApp)
+  await componentAbovemap3dStore.updateListLabelsStations(rennesApp)
+  componentAbovemap3dStore.addListenerForUpdatePositions(rennesApp)
 })
 
+async function filterFeaturesOnLine() {
+  await filterFeatureByParkingAndLine(rennesApp, lineViewStore.selectedLine)
+}
+
 poiStore.$subscribe(async () => {
```

Comment:
The poi filter doesn't work on 3D
![image](https://user-images.githubusercontent.com/121813008/217477495-d9aaef5c-881d-4d06-a6b3-47f57e88cfbf.png)